components/procore/actions/create-rfi/create-rfi.mjs

```diff
@@ -0,0 +1,135 @@
+import app from "../../procore.app.mjs";
+
+export default {
+  key: "procore-create-rfi",
+  name: "Create RFI",
+  description: "Create a new RFI. [See the documentation](https://developers.procore.com/reference/rest/rfis?version=latest#create-rfi).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    companyId: {
+      propDefinition: [
+        app,
+        "companyId",
+      ],
+    },
+    projectId: {
+      optional: false,
+      propDefinition: [
+        app,
+        "projectId",
+        ({ companyId }) => ({
+          companyId,
+        }),
+      ],
+    },
+    subject: {
+      type: "string",
+      label: "Subject",
+      description: "The Subject of the RFI",
+    },
+    questionBody: {
+      type: "string",
+      label: "Question Body",
+      description: "The Body of the Question",
+    },
+    rfiManagerId: {
+      propDefinition: [
+        app,
+        "rfiPotentialManagerId",
+        ({
+          companyId, projectId,
+        }) => ({
+          companyId,
+          projectId,
+        }),
+      ],
+    },
+    assigneIds: {
+      type: "string[]",
+      label: "Assignee IDs",
+      description: "The Assignee IDs of the RFI",
+      optional: false,
+      propDefinition: [
+        app,
+        "potentialAssigneeId",
+        ({
+          companyId, projectId,
+        }) => ({
+          companyId,
+          projectId,
+        }),
+      ],
+    },
+    reference: {
+      type: "string",
+      label: "Reference",
+      description: "The Reference of the RFI",
+      optional: true,
+    },
+    isPrivate: {
+      type: "boolean",
+      label: "Private",
+      description: "The Private status of the RFI",
```

Comment:
```suggestion
      description: "Whether the RFI is private or not",
```
Let's try to keep the descriptions consistent across props of the same app, there is a similar one in the next component that uses this pattern